text/0000-loop-break-value.md

```diff
@@ -0,0 +1,216 @@
+- Feature Name: loop_break_value
+- Start Date: 2016-05-20
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+(This is a result of discussion of
+[issue #961](https://github.com/rust-lang/rfcs/issues/961) and related to RFCs
+[352](https://github.com/rust-lang/rfcs/pull/352) and
+[955](https://github.com/rust-lang/rfcs/pull/955).)
+
+Let a `loop { ... }` expression return a value via `break my_value;`.
+
+# Motivation
+[motivation]: #motivation
+
+This pattern is currently hard to implement without resorting to a function or
+closure wrapping the loop:
```

Comment:
Maybe, and maybe it's a good thing. Sounds like it's time I un-learned to *never leave variables uninitialised*.

Points 3 and 4 still hold (the example above can be four words shorter), but this weakens the argument.